ext/dom/config.w32

```diff
@@ -16,27 +16,12 @@ if (PHP_DOM == "yes") {
 			entityreference.c \
 			token_list.c \
 			notation.c xpath.c dom_iterators.c \
-			namednodemap.c xpath_callbacks.c", null, "-Iext/dom/lexbor");
+			namednodemap.c xpath_callbacks.c", null, "/I ext/lexbor");
```

Comment:
@cmb69 Currently, the build fails with the following error:

```
ext\dom\parentnode\css_selectors.c(26): fatal error C1083: Cannot open include file: 'lexbor/selectors-adapted/selectors.h': No such file or directory
```

I installed the relevant header file below, but do you think I should add something like `-Iext/dom/` here? Please note that previously, the file was in `ext/dom/lexbor/lexbor/lexbor-adapted/selectors.h`, and now I removed one level from the path.

Reply:
P.S. I wanted to mean `-Iext/lexbor/` instead of `-Iext/dom/`